WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -45,16 +46,47 @@ public static void migrateAppWideMobileEditorPreferenceToRemote(final AccountSto
             return;
         }
 
-        // In a later version we might override mobile_editor setting if it's set to `aztec` and show a specific notice
-        // for these users ("We made a lot of progress on the block editor and we think it's now better than
-        // the classic editor, we switched it on, but you can change the configuration in your Site Settings").
-        // ^ This code should be here.
-
         // If the user is already in the rollout group, we can skip this the migration.
         if (AppPrefs.isUserInGutenbergRolloutGroup()) {
             return;
         }
 
+        // -------- Rollout: Phase 2 -------
+        //
+        // If the user as one Aztec enabled site, we'll migrate all his sites to gutenberg and show
```

Comment:
Nit: typo: s/as/has

Reply:
Updated in 4f4cdb2278e547207c78f2105bf0c4ce0d9b3e10